src/app/space-wizard/common/logger.ts

```diff
@@ -8,7 +8,7 @@ export interface ILogEntry {
   inner?: any;
 }
 export interface ILoggerDelegate {
-  (options: string | ILogEntry): void;
+  (options: string | ILogEntry,...args): void;
```

Comment:
it is accessed by several areas in the code ... the log delegate can be called in two ways
- as a string this.log('say something',optional-object-to-console-dir, optional-console.group)
or
- as an object this.log({message:'say something', warning:true},optional-object-to-dir, optional-console.groupEnd)